### DISCUSSION TITLE
Scroll spy refresh Bootstrap 5 problem

### DISCUSSION THREAD

**Discussion by aribudin** [twbs/bootstrap#31950] [General]
Hello, I found small problem in scrollspy Bootstrap 5
https://v5.getbootstrap.com/docs/5.0/components/scrollspy/#refresh

Uncaught TypeError: Cannot read property 'refresh' of null

demo : https://codepen.io/aribudin/pen/XWKNpZg

Scrollspy nav link not refresh after sreen resized

How to test ?
1. open demo and click in "progress" menu
2. resize desktop screen to small size
3. click in "home" menu
Scrollspy nav link still in "progress" menu

Have any idea ?

**Comment by XhmikosR** [comment]
```js
var scrollSpy = new bootstrap.ScrollSpy(document.body)

var dataSpyList = [].slice.call(
  document.querySelectorAll('[data-spy="scroll"]')
)

dataSpyList.forEach(function (dataSpyEl) {
  bootstrap.ScrollSpy.getInstance(dataSpyEl).refresh()
})
```

**Reply by aribudin** [reply]
thanks for your reply
but navlink active still not refreshing after sreen resized

How to test ?

1.open demo and click in "progress" menu
demo : https://codepen.io/aribudin/pen/XWKNpZg

2.resize desktop screen to small size ( mobile screen or tab screen )
3.click in "home" menu, but nav link still in "progress" menu

like this
http://prntscr.com/v7ytvf

**Reply by UnHumbleBen** [reply]
Sounds this bug: https://github.com/twbs/bootstrap/issues/32496
